flink-contrib/docker-flink/base/Dockerfile

```diff
@@ -38,12 +38,12 @@ ENV JAVA_HOME /usr/java/default/
 RUN echo 'root:secret' | chpasswd
 
 #SSH as root... probably needs to be revised for security!
-RUN sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/' /etc/ssh/sshd_config
```

Comment:
Can this be handled more flexibly using sed regular expressions such that any parameter is changed to `yes`?